src/config/authd-config.c

```diff
@@ -224,6 +231,11 @@ int Read_Authd(const OS_XML *xml, XML_NODE node, void *d1, __attribute__((unused
 
         config->force_options.enabled = legacy_force_insert;
     }
+    if (legacy_force_time > 0) {
```

Comment:
```suggestion
    if (legacy_force_time >= 0) {
```

`legacy_force_time` can be 0 as well. It is a valid read value that also has to be overwritten.

Reply:
Done